File: C#/begginer/1038.cs
using System; 

class URI {

  static void Main(string[] args) { 
    string[] input = Console.ReadLine().Split(' ');
    int code = int.Parse(input[0]);
    int amount = int.Parse(input[1]);

    double[] items = { 4.00, 4.50, 5.00, 2.00, 1.50 };

    double total = items[code - 1] * amount;

    Console.WriteLine($"Total: R$ {total:F2}");
  }

}

File: Java/begginer/1038.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);
        
        double[] price = { 4.00, 4.50, 5.00, 2.00, 1.50 };
        String[] input = scanner.nextLine().split(" ");
        
        int code = Integer.valueOf(input[0]).intValue();
        int quant = Integer.valueOf(input[1]).intValue();
        
        System.out.printf("Total: R$ %.2f\n", price[code - 1] * quant);
    }
 
}
